README.md

```diff
@@ -8,6 +8,91 @@ An implementation of [Amazon Ion Schema](http://amzn.github.io/ion-schema) in Ru
 
 **This package is considered experimental, under active/early development, and the API is subject to change.**
 
+## Getting Started
+
+The following rust code sample is a simple example of how to use this API. The `my_int_type` is an integer ion type. 
+A sample schema `my_schema.isl` can be loaded using `ion-schema-rust` as following:
+
+### Example schema `my_schema.isl`
+```
+schema_header::{
+  imports: [],
+}
+
+type::{
+  name: my_int_type,
+  type: int,
+}
+
+schema_footer::{
+}
+```
+
+### Simple rust example to demonstrate how to load a schema and validate an ion value
+```rust
+use ion_schema::authority::{DocumentAuthority, FileSystemDocumentAuthority};
+use ion_schema::external::ion_rs::value::owned::OwnedElement;
+use ion_schema::result::{ValidationResult, IonSchemaResult};
+use ion_schema::types::TypeRef;
+use ion_schema::schema::Schema;
+use ion_schema::system::SchemaSystem;
+use std::path::Path;
+use std::rc::Rc;
+
+fn main() -> IonSchemaResult<()> {
+    // Create authorities vector containing all the authorities that will be used to load a schema based on schema id
+    let document_authorities: Vec<Box<dyn DocumentAuthority>> = vec![Box::new(
+        FileSystemDocumentAuthority::new(Path::new("schema")), // provide a path to the authority base folder containing schemas
+    )];
+
+    // Create a new schema system from given document authorities
+    let mut schema_system = SchemaSystem::new(document_authorities);
+
+    // Provide schema id for the schema you want to load (schema_id is the schema file name here)
+    let schema_id = "my_schema.isl";
+
+    // Load schema
+    let schema: Rc<Schema> = schema_system.load_schema(schema_id)?;
```

Comment:
As a TODO, we really should have a public type that hides the `Rc` implementation detail.

Reply:
Yes, there is already an issue opened to track this: https://github.com/amzn/ion-schema-rust/issues/36